operator/docs/kit.cloudformation.yaml

```diff
@@ -1,36 +1,14 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Description: Resources used by https://github.com/awslabs/kit/operator
+Parameters:
+  ClusterName:
+    Type: String
+    Description: "Substrate/host cluster name"
 Resources:
-  KitNodeInstanceProfile:
-    Type: "AWS::IAM::InstanceProfile"
-    Properties:
-      InstanceProfileName: "KitNodeInstanceProfile"
-      Path: "/"
-      Roles:
-        - Ref: "KitNodeRole"
-  KitNodeRole:
-    Type: "AWS::IAM::Role"
-    Properties:
-      RoleName: "KitNodeRole"
-      Path: /
-      AssumeRolePolicyDocument:
-        Version: "2012-10-17"
-        Statement:
-          - Effect: Allow
-            Principal:
-              Service:
-                !Sub "ec2.${AWS::URLSuffix}"
-            Action:
-              - "sts:AssumeRole"
-      ManagedPolicyArns:
-        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEKSWorkerNodePolicy"
-        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEKS_CNI_Policy"
-        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
-        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonSSMManagedInstanceCore"
   KitControllerPolicy:
     Type: "AWS::IAM::ManagedPolicy"
     Properties:
-      ManagedPolicyName: "KitControllerPolicy"
+      ManagedPolicyName: !Sub "KitControllerPolicy-${ClusterName}"
```

Comment:
why would you need a `cluster-name` for the policy ?

Reply:
>this is the issue we were facing when there is an existing testbed in an account and you can't deploy another because CDK can't skip existing roles/policies.

We discussed offline, this isn't the issue we were facing earlier. It was for KitNodeInstanceRole as KitNodeInstanceProfile is unique.